pkg/tinyusb/hw/hw_esp32.c

```diff
@@ -39,3 +39,8 @@ int tinyusb_hw_init(void)
 
     return 0;
 }
+
+void phy_bbpll_en_usb(bool en)
```

Comment:
I'm wondering why this is needed for this board but was not needed for other ESP32-S2 boards?

Reply:
turns out it's not needed after all